### PR TITLE
unsubscribe implemented for stores

### PR DESCRIPTION
**Pull request by sripaulgit**
Using takeUtil to unsubscribe when components unmount. Possible fix for
#66.

### CODE REVIEW

**Comment by sripaulgit** [comment]
@Simply007 didn't mean to commit the loading part. Was experimenting that for next PR. Will remove that. Thanks!

**Comment by Simply007** [comment]
@sripaulgit 
nice!
Looking forward to the fix commit.

**Comment by sripaulgit** [comment]
@Simply007 Removed the unintended commit. Sorry about the delay. Looking forward to explore the HOC approach next. Thanks!

File: src/Pages/Contacts.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ class Contacts extends Component {
 
     componentWillUnmount() {
         CafeStore.removeChangeListener(this.onChange);
+		CafeStore.unsubscribe();
```
Comment (Simply007) [inline src/Pages/Contacts.js]: There are two tab characters instead of spaces

**Comment by Simply007** [comment]
If you have any question, feel free to ask @sripaulgit :ear:

**Pull request merged**